rockcraft.yaml

```diff
@@ -38,6 +38,14 @@ services:
     # working-dir: /opt/spark
     environment:
       SPARK_PROPERTIES_FILE: /etc/spark8t/conf/spark-defaults.conf
+  jupyter:
+    command: "spark-client.pyspark [ --username spark --namespace spark ]"
+    summary: "This is the Spark-powered Jupyter service"
+    override: replace
+    startup: disabled
+    environment:
+      PYSPARK_DRIVER_PYTHON: jupyter
+      PYSPARK_DRIVER_PYTHON_OPTS: "lab --no-browser --port=8888 --allow-root --ip=0.0.0.0 --NotebookApp.token='' --notebook-dir=/var/lib/spark/notebook"
```

Comment:
Is`--allow-root` needed? won't it break loadsa folks' security policies? Looks like it might?

Reply:
removed this flag